proj/hps_accel/gateware/stream.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, payload_type):
             ("ready", Shape(), DIR_FANIN),
             ("last", Shape(), DIR_FANOUT),
             ("payload", payload_type, DIR_FANOUT),
-        ])
+        ], src_loc_at=3)
```

Comment:
Such magic, wow

Reply:
(I went and looked in nmigen what `src_loc_at` does, and I get it, but its name is not very obvious!)

Reply:
src_loc_at tells the machinery how deep in the stack to look for a variable name in order to make a name for the Signal()s. It's only important when dumping VCD files or looking at the generated code.